README.md

```diff
@@ -14,74 +18,78 @@
 Backed by Y Combinator
 </div>
 
----
+# Fern
+
+Fern is an open source toolkit for designing, building and consuming REST APIs. With Fern, you can generate client libraries, documentation and boilerplate for your backend server in your chosen framework (e.g. FastAPI, Express).
```

Comment:
How about

> With Fern, you can generate client libraries, documentation and boilerplate for your backend server.

Reply:
To be fair, I think that would be a great description for the repository. In the readme though, I do want to emphasize Fern supports you through all stages of developing and maintaining an API a little bit more. Definitely the most succinct way to put it though!

Reply:
> With Fern, you can generate client libraries, documentation and boilerplate for your backend server.

I could see us mentioning REST APIs. 

Fern is an open source toolkit for REST APIs. With Fern, you can generate client libraries, documentation and boilerplate code for your backend server.